docs/diagrams/OverallActivityDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+
+start
+
+repeat : User opens Unibook;
```

Comment:
I think after opening UniBook, there should be a "user type in command" so that the more options arrow can return to that instead of returning it to "user open unibook". The current flow seems to imply that user has to reopen unibook if he wants to use another command.